overlord/snapstate/storehelpers.go

```diff
@@ -69,6 +70,28 @@ func userFromUserID(st *state.State, userIDs ...int) (*auth.UserState, error) {
 	return user, err
 }
 
+func refreshOptions(st *state.State, origOpts *store.RefreshOptions) (*store.RefreshOptions, error) {
+	var opts store.RefreshOptions
+
+	if origOpts != nil {
+		if origOpts.RequestSeed != "" {
+			// nothing to add
+			return origOpts, nil
+		}
+		opts = *origOpts
+	}
+
+	if err := st.Get("request-salt", &opts.RequestSeed); err != nil && err != state.ErrNoState {
```

Comment:
as a top level name,  "refresh-request-salt" or "refresh-salt" would be better  (we have other top level state names with (about) "refresh" in them)